pkg/backup/backup.go

```diff
@@ -200,7 +200,11 @@ func (kb *kubernetesBackupper) BackupWithResolvers(log logrus.FieldLogger,
 	backupRequest.ResourceIncludesExcludes = collections.GetResourceIncludesExcludes(kb.discoveryHelper, backupRequest.Spec.IncludedResources, backupRequest.Spec.ExcludedResources)
 	log.Infof("Including resources: %s", backupRequest.ResourceIncludesExcludes.IncludesString())
 	log.Infof("Excluding resources: %s", backupRequest.ResourceIncludesExcludes.ExcludesString())
-	log.Infof("Backing up all pod volumes using restic: %t", *backupRequest.Backup.Spec.DefaultVolumesToRestic)
+	if backupRequest.Backup.Spec.DefaultVolumesToRestic != nil {
```

Comment:
There is [a utility function](https://github.com/vmware-tanzu/velero/blob/main/pkg/util/boolptr/boolptr.go#L20) that we can use to determine the value of a boolean pointer.

Reply:
Would you prefer the following over what I have? 

```go
if boolptr.IsSetToTrue(backupRequest.Backup.Spec.DefaultVolumesToRestic) || boolptr.IsSetToFalse(backupRequest.Backup.Spec.DefaultVolumesToRestic) {
    log.Infof("Backing up all pod volumes using Restic: %t", *backupRequest.Backup.Spec.DefaultVolumesToRestic)
} else {
    log.Infof("DefaultVolumesToRestic for backing up all pod volumes using Restic is %v", backupRequest.Backup.Spec.DefaultVolumesToRestic)
}
```

Another alternative is add a Print() function to the utility function. Something like,
```go
package boolptr
...
// Print prints the value of the bool pointer.
func Print(b *bool) string {
    if b == nil {
        return "<nil>"
    } else {
        return fmt.Sprint("%t", *b)
    }
}
```

I'm liking adding another utility function, Print. What do you think @ywk253100?

Reply:
The `nil` value of `backupRequest.Backup.Spec.DefaultVolumesToRestic` can be regarded as `false`. I saw some other similar usages for the function, e.g. https://github.com/vmware-tanzu/velero/blob/main/pkg/backup/item_backupper.go#L145

Reply:
I didn't want to assume `nil` was the same as `false`, but I think you're right--here it can be treated as false.

If `nil` is always treated the same as `false`, I don't think a boolean pointer is needed; we could just use a boolean variable. But I think that is outside the scope of the reported issue.

I have made the change to use the boolptr utility function and removed the if/else blocks. Now, only a single line is used for logging.